### PR TITLE
Metropolis: static call

### LINKED ISSUE #234: Adding STATIC_CALL opcode
This issue keeps track of following the adoption of https://github.com/ethereum/EIPs/pull/214/files

### PR DESCRIPTION
**Pull request by pirapira**
This PR introduces https://github.com/ethereum/EIPs/pull/214 and fixes #234 as part of the Metropolis changes.

This is a reincarnation of #237.

### CODE REVIEW

File: Paper.tex [hunk 1]
```diff
@@ -826,7 +828,10 @@ \subsection{Execution Environment}
 \item $I_v$, the value, in Wei, passed to this account as part of the same procedure as execution; if the execution agent is a transaction, this would be the transaction value.
 \item $I_\mathbf{b}$, the byte array that is the machine code to be executed.
 \item $I_H$, the block header of the present block.
-\item $I_e$, the depth of the present message-call or contract-creation (i.e. the number of {\small CALL}s or {\small CREATE}s being executed at present).
+\item $I_e$, the depth of the present message-call or contract-creation
+      (i.e. the number of {\small CALL}s or {\small CREATE}s being
```
Comment (gavofyork) [inline Paper.tex]: spurious new lines.

File: Paper.tex [hunk 2]
```diff
@@ -826,7 +828,10 @@ \subsection{Execution Environment}
...
+      executed at present).
+\item $I_w$, the permission to write to the state.
```
Comment (gavofyork) [inline Paper.tex]: "...to alter make modifications to the state" sounds a little more natural

**Comment by pirapira** [comment]
Addressed the comments.

**Comment by pirapira** [comment]
I should target `master`.  Because `master` is the Byzantium version now, where static call should be introduced.

**Pull request closed without merging**